### ISSUE TITLE
tests fail in iTerm2  2.9.20160313

### ISSUE THREAD

**Issue by ashander** [sindresorhus/iterm2-version#1]
``` sh
git clone https://github.com/sindresorhus/iterm2-version.git
cd iterm2-version
npm install
npm test
```

``` sh
   1 failed

  1. [anonymous]
  failed with "Command failed: ./main iTerm"
      checkExecSyncError (child_process.js:469:13)
    Object.execFileSync (child_process.js:489:13)
    Function.module.exports.module.exports.sync.app [as sync] (node_modules/app-path/index.js:40:25)
    module.exports (index.js:11:32)
    Test.fn (test.js:10:18)

npm ERR! Test failed.  See above for more details.
```

on OS X 10.10.5 (14F27) and iTerm2 Build 2.9.20160313

```
{ 'iterm2-version': '2.1.0',
  npm: '3.6.0',
  ares: '1.10.1-DEV',
  http_parser: '2.6.2',
  icu: '56.1',
  modules: '47',
  node: '5.7.1',
  openssl: '1.0.2g',
  uv: '1.8.0',
  v8: '4.6.85.31',
  zlib: '1.2.8' }
```

**Comment by sindresorhus** [comment]
Can you try running the tests in https://github.com/sindresorhus/app-path? I have a feeling it's not this module.

**Comment by ashander** [comment]
Tests in app-path work, with one warning:

```
~/code/npm-collab/app-path (master)$ npm test

> app-path@2.1.0 test ~/code/npm-collab/app-path
> xo && ava


~/code/npm-collab/app-path/index.js
  39:3  warning  Unexpected 'todo' comment  no-warning-comments

✖ 1 problem (0 errors, 1 warning)

   4 passed
```

**Comment by sindresorhus** [comment]
Weird. Maybe try updating to the latest iTerm version? (Build 2.9.20160322)

Can you install https://github.com/sindresorhus/app-path-cli and run `$ app-path iTerm` and post the result?

**Comment by ashander** [comment]
Latest on https://www.iterm2.com/downloads.html is the one I have
(Build 2.9.20160313). I'm assuming your later version number is the nightly?

I'll download it and see. And also try the CLI

**Comment by ashander** [comment]
Lol, I think this is it. My app is called iTerm2. 

``` sh
~/code/npm-collab$ ./node_modules/.bin/app-path iTerm2
/Applications/iTerm2.app
~/code/npm-collab$ ./node_modules/.bin/app-path iTerm
Error: Couldn't find the app
    at ChildProcess.exithandler (child_process.js:202:12)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at Pipe._onclose (net.js:475:12)
~/code/npm-collab$ ./node_modules/.bin/app-path iTerm
```

**Comment by ashander** [comment]
Confirmed, still in build 2.9.20160313, but with the app bundle renamed to `/Applications/iTerm.app`, the tests pass

``` sh
> iterm2-version@2.1.0 test ~/code/npm-collab/iterm2-version
> xo && ava


   1 passed
```

**Comment by ashander** [comment]
![screen shot 2016-03-22 at 10 15 35 pm](https://cloud.githubusercontent.com/assets/510776/13976292/b33c38ca-f07b-11e5-8a39-1dc62626fb40.png)

**Comment by sindresorhus** [comment]
Why were your iTerm app named `iTerm2`? Something you did or are older versions named differently?

**Comment by ashander** [comment]
No, I did it. I installed the latest (unstable 2.9.20160313 beta) to try
out your term-img module
and because I installed it alongside my older version, I named it iTerm2

**Comment by sindresorhus** [comment]
I see, but it should still have found your older version though. Right?

**Comment by ashander** [comment]
Yes, you're right. Sorry, but I hadn't looked carefully and my old version (2.1.4) was name `iTerm 2.app`, with a space. I must have done that too, when I installed it, because I just checked and it comes as `iTerm.app` from the install bundle (iTerm2-2_1_4.zip). So, when the 'bug' occurred, I had iTerm2.app and iTerm 2.app but no iTerm.app.

Thanks for all your node modules btw, and for helping with this.
